python/lsst/afw/geom/detail/frameSetUtils.cc

```diff
@@ -0,0 +1,71 @@
+/*
+ * LSST Data Management System
+ * See COPYRIGHT file at the top of the source tree.
```

Comment:
It is now allowed, and even encouraged, but afw has no such file so I changed the above.